src/Aquifer.Jobs/Subscribers/NotificationSubscriber.cs

```diff
@@ -0,0 +1,174 @@
+                project.Id,
+                companyLeadUser.Id);
+        }
+        else
+        {
+            _logger.LogWarning(
+                "Unable to send project started notification for Project ID {ProjectId} because there is no assigned Company or Company Lead.",
+                project.Id);
+        }
+    }
+
+    [Function(nameof(SendResourceCommentCreatedNotification))]
+    public async Task SendResourceCommentCreatedNotification(
+        [QueueTrigger(Queues.SendResourceCommentCreatedNotification)] QueueMessage queueMessage,
+        CancellationToken ct)
+    {
+        var message = queueMessage.Deserialize<SendResourceCommentCreatedNotificationMessage, NotificationSubscriber>(_logger);
+
+        var dbConnection = _dbContext.Database.GetDbConnection();
+        var resourceCommentData = await GetResourceCommentDataAsync(dbConnection, message.CommentId, ct);
+
+        // Get users who are assigned or have previously been assigned to this resource
+        // and who are in the same company as the user who made the comment,
+        // excluding the user who made the comment (who should not get a notification).
+        var previouslyAssignedUsers = await _dbContext.ResourceContentVersionAssignedUserHistory
+            .Where(rcvauh =>
+                rcvauh.ResourceContentVersion.ResourceContentId == resourceCommentData.ResourceContentId &&
+                rcvauh.AssignedUserId != null &&
+                rcvauh.AssignedUserId.Value != resourceCommentData.UserId)
+            .Select(rcvauh => rcvauh.AssignedUserId)
+            .Distinct()
+            .Join(
+                _dbContext.Users
+                    .Where(u => u.Id == resourceCommentData.UserId && u.AquiferNotificationsEnabled)
+                    .SelectMany(u => u.Company.Users),
+                userId => userId,
+                u => u.Id,
+                (_, u) => u)
+            .ToListAsync(ct);
+
+        if (previouslyAssignedUsers.Count == 0)
+        {
+            _logger.LogInformation(
+                "Comment ID {CommentId} was added to Resource Content ID {ResourceContentId} by User ID {CommenterUserId} but no email was sent because only the commenter has previous resource assignment.",
+                resourceCommentData.CommentId,
+                resourceCommentData.ResourceContentId,
+                resourceCommentData.UserId);
+            return;
+        }
+
+        var commenterUser = await _dbContext.Users
+            .SingleAsync(u => u.Id == resourceCommentData.UserId, ct);
+
+        var templatedEmail = new TemplatedEmail(
+            From: NotificationsHelper.NotificationSenderEmailAddress,
+            // Template Designer: https://mc.sendgrid.com/dynamic-templates/d-ea84b461ed0f439589098053f8810189/version/26393e82-7d17-4b8f-a37c-cb20f62d8802/editor
+            TemplateId: "d-ea84b461ed0f439589098053f8810189",
```

Comment:
See https://biblionexusworkspace.slack.com/archives/C05K5N7F44Q/p1729881033556149 for design review of this email template.